app/src/main/res/values/strings.xml

```diff
@@ -101,6 +101,7 @@
 
     <string name="my_schedule_empty">No items here. Let\'s check out all sessions and put your favourite!</string>
     <string name="my_schedule_go_to_session_list">Go to All Sessions</string>
+    <string name="my_schedule_conflicted">Conflict happens</string>
```

Comment:
This label sounds a bit weird to me. Could you change it to "Conflicted with another session" or something?